android/pom.xml

```diff
@@ -19,6 +19,13 @@
     <maven-javadoc-plugin.version>3.1.0</maven-javadoc-plugin.version>
     <maven-source-plugin.version>3.2.0</maven-source-plugin.version>
     <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+    <module.status>integration</module.status>
+    <module.parentVersion>HEAD-jre-SNAPSHOT</module.parentVersion>
```

Comment:
> I don't think that's the case: I see no <parent> element in android/pom.xml.

Uh, my bad. Yes the POM does not have the parent. And while it could be possible it does make sense that there is no common parent as each flavor is a separate maven build.

I don't know why I did this, but this `parentVersion` property is actually not needed at all and the version of the parent simply has to match the version of the variant. Then the `checker-compat-qual` version is also there.

So forget most of what I said earlier.

I fixed this. Thanks for clarifying and asking the right questions.

Reply:
Ah, thanks, I was still not seeing the implications of this :) All the more reason for the Gradle metadata to simplify things for users!